web/new_playground.html

```diff
@@ -54,7 +51,8 @@
 </head>
 
 <body>
-<header>
+
+<header class="mdc-elevation--z4">
```

Comment:
nit: It looks like this change is applying a box shadow to the header, which (while more attractive in general) is overlapping the Flutter output. While most people won't notice or be bothered by the shadow, it might be better to have as pristine a presentation of the flutter code as possible. If I were playing with a top-of-the-screen LinearProgressIndicator or something, I can see preferring it that way.